deps/wazuh_testing/wazuh_testing/tools/file.py

```diff
@@ -236,7 +236,7 @@ def delete_file(file_path):
 
 def delete_path_recursively(path):
     if os.path.exists(path):
-        shutil.rmtree(path, onerror=on_write_error)
+        shutil.rmtree(path, ignore_errors=True, onerror=on_write_error)
```

Comment:
Why do you ignore the errors?

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/3906/commits/35f4ac6330db8b957231daa9f2131cf30eeaab0c